requirements_dev.txt

```diff
@@ -2,4 +2,5 @@ mock
 httpretty
 pytest
 pytest-cov
-sphinx
\ No newline at end of file
+sphinx
+prometheus_client
```

Comment:
How does this relate to https://github.com/blue-yonder/postgraas_server/blob/master/requirements_prometheus.in?

Reply:
Same purpose, sorry I didn't see that file